web/src/components/index/indexFeatures.js

```diff
@@ -95,9 +95,9 @@ export default function IndexFeatures({ content }) {
                     {features.map((feature) => {
                         return (
                             <GridItem
-                                key={feature.id}
-                                onClick={() => setActiveTab(feature.id)}
-                                isActiveTab={activeTab === feature.id}
+                                key={features.id}
+                                onClick={() => setActiveTab(features.id)}
+                                isActiveTab={activeTab === features.id}
```

Comment:
I there a reason this is changed? Doesn't seem like it should be part of the work on the 404 page but maybe I'm missing something.

Also, I think it breaks the functionality. We're mapping over the `features` array using each `feature` item to return some jsx. `features` doesn't really exist inside the `map()` method.